tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -2212,3 +2213,59 @@ def benford_correlation(x):
     # np.corrcoef outputs the normalized covariance (correlation) between benford_distribution and data_distribution.
     # In this case returns a 2x2 matrix, the  [0, 1] and [1, 1] are the values between the two arrays
     return np.corrcoef(benford_distribution, data_distribution)[0, 1]
+
+
+@set_property("fctype", "combiner")
+def query_similarity_count(x, param):
+    """
+    This feature calculator accepts an input query subsequence parameter,
+    compares the query (under z-normalized Euclidean distance) to all
+    subsequences within the time series, and returns a count of the number
+    times the query was found in the time series (within some predefined
+    maximum distance threshold).
```

Comment:
Maybe mention, that users need to enable this feature by themselves as the default implementation will always return NaN.

Reply:
Good idea. Thank you